scripts/setup_hooks.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+git config core.hooksPath "./hooks"
```

Comment:
Hi @shaman-rajan -- I checked with @rabidbit, and I think it's fine to just add this to the main setup script. So then we would tell devs: run the main setup script on your first push, and then thereafter you just need to "git push" and not run anything else (incl. tests). We can also add a bypass --disable-presubmit-checks flag too. How does that sound?